src/cosalib/aws.py

```diff
@@ -0,0 +1,19 @@
+import boto3
+from cosalib.cmdlib import (
+    retry_stop,
+    retry_boto_exception,
+    retry_callback
+)
+from tenacity import retry
+
+
+@retry(stop=retry_stop, retry=retry_boto_exception, before_sleep=retry_callback)
+def deregister_ami(ami_id, region):
```

Comment:
If we're going to grow a lot of these, I think we could auto-generate them.  But fine as is now.

Reply:
I don't think we'll have more than a few here. If that list grows large `ore` should be updated instead

Reply:
OK, fair enough.  And right...this is one of many "mantle vs cosa" issues.